godot_ap/autoloads/archipelago.gd

```diff
@@ -108,19 +115,21 @@ class ComplexColor:
 enum RichColor {
 	NIL, RED, GREEN, YELLOW, BLUE,
 	MAGENTA, CYAN, WHITE, BLACK, SLATEBLUE,
-	PLUM, SALMON, ORANGE, GOLD,
+	PLUM, SALMON, ORANGE, GOLD
 }
 enum SpecialColor {
-	PLAYER, ITEM_PROG, ITEM, ITEM_USEFUL, ITEM_TRAP,
+	ANY_PLAYER, OWN_PLAYER, ITEM_PROG, ITEM, ITEM_USEFUL, ITEM_TRAP, ITEM_PROGUSEFUL,
 	LOCATION, UI_MESSAGE, DEBUG
 }
```

Comment:
When adding to enums in godot (at least at the moment), you should probably add to the end, as if any `@export` variable is using that enum as a type, adding things not at the end will change the value of all those exports. (Yes, this kinda feels like it defeats the purpose of enums, pretty sure there's already godot proposals to fix this in various ways, it's just an issue right now).

This said, the color handling here was just added, so, it's fine this time imo; just commenting so you're aware going forward that this can cause issues in some cases.